web/templates/web/faq.html

```diff
@@ -194,7 +195,7 @@ <h2 class="accordion-header" id="heading-9">
                  data-bs-parent="#faq-accordion-participation">
                 <div class="accordion-body">
                     {% blocktranslate %}
-                <p>Rather than having the parent or legal guardian sign a form, we ask that you read aloud (or sign in ASL) a statement of consent which is recorded using your webcam. This statement holds the same weight as a signed form, but should be less hassle for you. It also lets us verify that you understand written English and that you understand you're being videotaped.</p>
+                <p>Some studies ask you (the parent or guardian) to sign an online form after either reading about what the study involves or talking with a researcher. Other studies, especially ones that happen on our own Children Helping Science platform, ask that your read aloud (or sign in ASL) a statement of consent which is recorded using your webcam. This statement holds the same weight as a signed form, but should be less hassle for you. It also lets us verify that you understand written English and that you understand you're being videotaped.</p>
```

Comment:
Seems like there's a typo here: "Other studies ... ask that your read aloud ... a statement of consent." I think "your" should be "you"? 
BTW I realize that this is the exact change that @mekline requested [here](https://github.com/lookit/lookit-api/issues/1179#issuecomment-1529937276) so I'm suggesting that the typo is from there.